server.js

```diff
@@ -26,7 +25,7 @@ app.use(webpackHotMiddleware(compiler));
 
 const fractal = Fractal.create();
 fractal.components.set('path', path.join(__dirname, 'src/components'));
-fractal.components.set('ext', '.html');
+fractal.components.set('ext', '.dust');
```

Comment:
Is there any way we could also support `hbs` as well?

Reply:
fractal only allows on templating language